README.md

```diff
@@ -1,18 +1,80 @@
-# Hang In There
+# Hang In There!
 
-A boilerplate repo. 
+## Table of Contents
+- [Introduction](#introduction)
+- [Technologies](#technologies)
+- [Set Up](#set-up)
+- [Deployment Link](#deployment-link)
+- [Features](#features)
+- [Reflection](#reflection)
+- [Future Features](#future-features)
+- [Contributors](#contributors)
+- [Project Specifications](#project-specifications)
 
-## Set Up
+### Introduction
+We all need a little motivation in our lives and that’s what motivational posters are for! But have you ever wanted to make motivational posters tailored to you and your specific needs for inspiration?
 
-1. One teammate: fork this repository
-2. Clone down your new, forked repo
-3. cd into the repository
-4. Open it in your text editor
-5. Add all project partners and your assigned instructor as collaborators on the repository
+Well now with Hang In There you can! Use our Make Poster form to create posters with titles, quotes, and images chosen by **YOU**. No idea what motivation you need? Let our curated list of inspirational images, quotes, and titles make **YOUR** motivation. You can even save your favorite motivational posters to view them later.
 
-Project spec & rubric can be found [here](https://frontend.turing.io/projects/module-1/hang-in-there.html)
+### Technologies
+- JavaScript
+- HTML
+- CSS
 
-To view your project:
+### Set Up
+1.  Clone this [repository](https://github.com/davidhdaw/hang-in-there-project).
+2. `cd` into the directory.
+3. Run `open index.html`.
 
-1. In your terminal, navigate to your project repo
-2. Run the command `open index.html`
+### Deployment Link
+- Simply go to: https://davidhdaw.github.io/hang-in-there-project/
+
+### Features
+- User can view a random poster on page load
+- User can view a random poster each time _Show Another Random Poster_ button is clicked
+
+![random poster gif](https://media.giphy.com/media/5KBlIu9PnA8s6Ykzpr/giphy.gif)
```

Comment:
Thank you for these gifs they are great literal illustrations of our functionality.